pkg/azure/aks/machine_store.go

```diff
@@ -147,22 +149,15 @@ func (m *MachineStore) getVmInfoFromResourceGroup(ctx context.Context, rgName, c
 
 		for _, v := range nextResult.Value {
 			vmssName := to.String(v.Name)
-			vmssPriority := getMachineScaleSetPriority(v)
-			vmssOperationSystem := getMachineScaleSetOperatingSystem(v)
-
-			vmInfo, err := m.getVmInfoFromVmss(ctx, rgName, vmssName, vmssPriority, vmssOperationSystem)
-			if err != nil {
-				return nil, err
+			if len(vmssName) == 0 {
+				m.logger.Error("unable to determine VMSS name")
```

Comment:
Is there anything we can log here to help us identify what exactly what is being skipped? Not sure if there's any value in logging this out as we still add it to the array, though maybe that was an oversight?

Reply:
This was indeed an oversight.  Updating.